users/models.py

```diff
@@ -242,6 +242,88 @@ def has_quiz(self) -> bool:
     def has_user_card(self) -> bool:
         return hasattr(self, "user_card")
 
+    @property
+    def quizs_answer_count(self) -> int:
```

Comment:
```suggestion
    def quiz_answer_count(self) -> int:
```
idem pour les autres :sweat_smile:

Reply:
Corrigé